sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1559,6 +1559,17 @@ object RapidsConf {
     .integerConf
     .createWithDefault(value = 0)
 
+  val CONCURRENT_WRITER_PARTITION_FLUSH_SIZE =
+    conf("spark.rapids.sql.concurrentWriterPartitionFlushSize")
+        .doc("The flush size of the concurrent writer cache in bytes for each partition. " +
+            "If specified spark.sql.maxConcurrentOutputFileWriters, use concurrent writer to " +
+            "write data. Concurrent writer first caches data for each partition and begins to " +
+            "flush data if find one partition data is equal or greater than this size. " +
+            "Max value may get better performance but not always, because concurrent writer uses" +
+            "spillable cache and big value may cause more IO swaps")
+        .bytesConf(ByteUnit.BYTE)
+        .createWithDefault(ByteUnit.MiB.toBytes(64))
```

Comment:
Should we set this to 128 MiB? or perhaps see if we can figure out the target row group size is in parquet and target stripe size is in ORC as a default?

Reply:
Now, read from the SQLConf